pymobiledevice3/remote/xpc_message.py

```diff
@@ -152,6 +153,10 @@ def _decode_xpc_data(xpc_object) -> bytes:
     return xpc_object.data
 
 
+def _decode_xpc_date(xpc_object) -> datetime:
+    return datetime.fromtimestamp(xpc_object.data / 1000000000)
```

Comment:
Hard to count the 0, maybe add comment "Convert from nanoseconds to seconds"

Reply:
done